js/bank.js

```diff
@@ -4,118 +4,113 @@ var rl = readline.createInterface({
   output: process.stdout
 });
 
-function start(balance) {
+
+function bank(balance) {
```

Comment:
I like how simple and small this is.  I think a better name would be `run` , because it's the function in charge of running your program.

Reply:
Ok I can change the name to run